R/usage_to_annual_nets_distributed.R

```diff
@@ -0,0 +1,234 @@
+#' Convert bednet usage to annual nets distributed per capita
+#' 
+#' Assumptions: 
+#' - Net use rates stays at 2019 levels for all countries
+#' - When converting from access to NPC, use the 2020 loess curve fit of access-NPC 
+#'   at the country-month level.
+#' 
+#' @param target_usage A single value or vector of desired target usages to model.
+#' @param country_iso3 Vector of ISO3 codes of African countries for which to return results. 
+#' Default = "all".
+#' @param extrapolate_npc Option to extrapolate target nets per capita beyond fitted 
```

Comment:
For the extrapolation, I think we want the trend from the end of curve (say the last 4 points) to be extrapolated.

Currently if I run:
```
p1 <- find_annual_nets_distributed(seq(0, 1, 0.01), country_iso3 = "NGA")
plot(p1$annual_percapita_nets_distributed ~ p1$target_use,
     xlab = "Target Use", ylab = "Annual nets distributed")
```
I get:
![image](https://user-images.githubusercontent.com/16702715/151959307-81250258-ad9e-4b40-8e70-314bc014ddb5.png)

This makes it get more efficient at the end, which I don't think would happen

Reply:
I included this unrealistic option in case you wanted to reach an access of 1 (so you can run higher usages) but let me know if you think its better to remove altogether. Option "loess" now added is more realistic